inst/examples/installGithub.r

```diff
@@ -13,8 +13,9 @@ suppressMessages({
 })
 
 ## configuration for docopt
-doc <- "Usage: installGithub.r [-h] [-x] [-d DEPS] [-u UPDATE] [REPOS...]
+doc <- "Usage: installGithub.r [-r DEPREPO...] [-h] [-x] [-d DEPS] [-u UPDATE] [REPOS...]
```

Comment:
And I think the three dots here have special meaning ... which we do not want.  

Also, as I have been picking so much, ~~two~~three more changes:
- can we renamed it to CRAN rather than DEPREPO ?  Shorter, simpler, obvious default
- can you place it behind `[-u UPDATE]` here and below it below?
- can you add an entry to ChangeLog?